packages/api/src/schema/schema.yaml

```diff
@@ -1426,6 +1438,12 @@ components:
           example: true
         playbackPolicy:
           $ref: "#/components/schemas/playback-policy"
+        creatorId:
+          oneOf:
+            - $ref: "#/components/schemas/asset/properties/creatorId"
+            - type: string
+              description:
+                Developer-managed (unverified) creator ID of the asset
```

Comment:
Wonder if we need this helper syntax [1] or if it would be better to have developers explicitly specify the `"unverified"` string just so it's on their face lol

[1] (this means that when creating an asset one could not only specify `creatorId: {type:unverified, value: foo}` but also just `creatorId: foo`)

Reply:
Yeah, I like the helper syntax. Especially since we only support `unverified` currently, it seems clunky to require that to be passed in.